### DISCUSSION TITLE
The Mania touchscreen experience on Android is very poor

### DISCUSSION THREAD

**Discussion by X-X-X-X-X-X-X-X-X-X-X-X-X** [ppy/osu#30769] [Ideas] [answered]
![1f25baf4c7e7745f996c357ec7af87a0](https://github.com/user-attachments/assets/d3e35217-94c2-4b06-b5c3-901c5d37196a)

This virtual button is very obstructing the view and has a very poor sense of impact, and there is also a pause button in the bottom right corner of the interface that is very easy to accidentally touch, which seems to be completely unsuitable for touchscreen players.

I suggest referring to the game interface of other drop in music games, perhaps like this:
1. Add a mania mode track width automatic screen filling option, which can eliminate virtual buttons
2. Move the pause button below PP

这虚拟按钮非常遮挡视野并且打击感非常糟糕，而且界面右下角还有暂停按钮非常容易误触，这看起来似乎完全不是为触屏玩家准备的。

我建议参考下其他下落式音游的游戏界面，或许可以这样：
1. 新增一个mania模式轨道宽度自动铺满屏幕选项，这样就可以抛弃虚拟按键
2. 将暂停按钮移动至pp下方

**Comment by Xionghaizi001** [comment]
Another suggestions for it:
- Make the button easier to corresponding with actual keys. For example to change the color of button, or just display a number of every tracks.

另一个非常重要的建议：
- 尽可能让触摸按钮与实际轨道对上。例如可以把按钮的颜色和轨道同步，或者干脆显示个数字在按钮上，这样就可以很方便地知道哪个按钮对应哪个轨道。

**Comment by peppy** [comment, marked as answer]
The columns used to expand to the screen size, but people didn't like this.

We'll add more customisation in the future.

**Reply by X-X-X-X-X-X-X-X-X-X-X-X-X** [reply]
Perhaps they are not mobile gamers.

Hope to release the options as soon as possible, this is important, thank you.

**Comment by DarkGradient** [comment]
Really, peppy, I beg you, make tiles expand on fullscreen.
It's horrible to play mania on phone.
